config/initializers/extends.rb

```diff
@@ -450,12 +450,24 @@ class Extends
     task_link_sharing_enabled: 246,
     task_link_sharing_disabled: 247,
     shared_task_message_edited: 248,
-    result_content_rearranged: 244
+    result_content_rearranged: 249,
+    result_table_added: 250,
+    result_file_added: 252,
+    result_file_deleted: 253,
+    result_text_added: 254,
+    result_text_edited: 255,
+    result_text_deleted: 256,
+    result_table_edited: 257,
+    result_table_deleted: 258,
+    result_duplicated: 259,
+    result_text_duplicated: 260,
+    result_table_duplicated: 261,
+    restore_result: 262
```

Comment:
we should call this `result_restored`, to keep it consistent with all the other result activity names